.circleci/config.yml

```diff
@@ -56,11 +57,12 @@ jobs:
             - ./retraced.tar
 
   build_postgres:
-    <<: *defaults
+    docker:
+      - image: cimg/base:2023.12
     steps:
       - checkout
       - setup_remote_docker:
-          version: 19.03.13
+          version: 20.10.14
```

Comment:
This change is recommended by Circle to resolve Debian repo errors

https://support.circleci.com/hc/en-us/articles/6025804632987-How-to-resolve-The-following-signatures-couldn-t-be-verified-because-the-public-key-is-not-available-when-building-a-docker-image